scmtfvc.properties

```diff
@@ -2,14 +2,20 @@ category=Integration
 description=Provides SCM TFVC integration
 devVersion=
 privateVersions=
-publicVersions=2.2
+publicVersions=2.3,2.2
```

Comment:
Sorry, I didn't catch the first time that you weren't archiving the old version. In fact, I need you to archive the old version because the compatibility of public versions can't overlap. Normally, you'd set the end range of 2.2 to before 2.3 compatibility starts, but that puts it ending at 7.8 and _that_ SQ version has been archived. So... the upshot is that you need to move 2.2 to `archivedVersions`.

Reply:
done